src/components/helpers/dom_helpers.ts

```diff
@@ -18,3 +20,16 @@ export function gridOverlayPosition() {
 export function getOpenedMenus(): HTMLElement[] {
   return Array.from(document.querySelectorAll<HTMLElement>(".o-spreadsheet .o-menu"));
 }
+
+function isMacOS(): boolean {
+  return Boolean(navigator.userAgent.match(macRegex));
```

Comment:
`return macRegex.test(navigator.userAgent);`
not that it really matters here, but `test` is faster than `match`

Reply:
actually, the same helper was also merged by https://github.com/odoo/o-spreadsheet/pull/3229/files

just needs a small rebase ^^